templates/projects.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
+
+{% block meta_title %}Projects{% endblock meta_title %}
+
+{% block main_content %}
+
+    <h4>Projects list:</h4>
+
+    <table class="table table-hover table-striped">
+        <thead>
+            <tr>
+                <th> id </th>
+                <th> title </th>
+                <th> members number </th>
+                <th> members list </th>
+                <th> start date </th>
+                <th> end date </th>
+        </thead>
+        <tbody>
+        {% for project in projects %}
+
+            <tr>
+                <td>{{ project.pk }}</td>
+                <td><a href="{% url "project_detail" project.pk %}">{{ project.title }}</a></td>
+                <td>{{ project.relation.all.count }}</td>
+                <td>
+                    <div class="dropdown">
+                        <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown"> watch
+                        <span class="caret"></span></button>
+                        <ul class="dropdown-menu">
+                            {% for member in project.relation.all %}
+                                <li><a href="{% url "profile" member.member.pk %}">{{ member.member.username }}</a></li>
```

Comment:
Member is also a different table, so you need to prefetch(or maybe select_related in this case?) it as well :)